src/virtual-machine.js

```diff
@@ -366,7 +366,11 @@ class VirtualMachine extends EventEmitter {
         const vm = this;
         const promise = storage.load(storage.AssetType.Project, id);
         promise.then(projectAsset => {
-            vm.loadProject(projectAsset.data);
+            if (!projectAsset) {
+                log.error("Failed to fetch project with id: ", id);
```

Comment:
lint error here with the double-quotes
```suggestion
                log.error(`Failed to fetch project with id: ${id}`);
```

Reply:
oh oops! I thought I ran the tests again but I think I probably added that change later